eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,48 @@ $packagePattern = "*.zip"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=python%2F&delimiter=%2F"
 
+function Get-AllPackageInfoFromRepo
+{
+  $allPackageProps = @()
+  Push-Location $RepoRoot
+  $allSetupProps = $null
+  try{
+    pip install packaging==20.4 -q -I
+    $allSetupProps = (python -c "import sys; import os; sys.path.append(os.path.join('eng', 'scripts')); \
+    import get_package_properties; get_package_properties.get_all_package_properties()")
+  }
+  catch
+  {
+    # This is soft error and failure is expected for python metapackages
+    LogError "Failed to get all package properties"
+  }
+  Pop-Location
+  $allMatches = $allSetupProps | Select-String '\(.*?\)' -AllMatches
```

Comment:
Wouldn't these already be one entry per line? In which case you can just loop over them and then split on ",", if not we probably should make our python script do that.

Reply:
No, its just one giant string